cmd/sqlcmd/sqlcmd.go

```diff
@@ -117,7 +121,15 @@ func (a SQLCmdArguments) authenticationMethod(hasPassword bool) string {
 	return a.AuthenticationMethod
 }
 
+func initLocale() {
+	localeName := os.Getenv("SQLCMD_LOCALE")
```

Comment:
>os.Getenv("SQLCMD_LOCALE") [](http://example.com/codeflow?start=15&length=26)

it's not a locale we are getting, it's a language.
Locale is about formatting based on regional settings.

how about SQLCMD_LANGID or SQLCMD_LANGUAGE
We could potentially support LCIDs too

Reply:
are there os-specific methods to get the user's current preferred language? Windows users can use different UI languages than the OS language.
The ODBC sqlcmd only installs 1 language, IIUC. We'll be installing all languages so we should be able to use a fallback logic that's aware of the Windows MUI model.

Reply:
Currently we expect BCP47 tag in lowercase but can be extended to LCIDs too. 
I did not consider windows MUI model. Will look into the window's specific APIs for the same if any.